src/components/Clusters/ClusterConfigCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { Badge, Button, Card, Text } from '@mantine/core';
+import { format } from 'date-fns';
+import { RiSettings4Line } from 'react-icons/ri';
+
+import { useAuth } from '@/hooks/useAuth';
+import type { Cluster } from '@/types/cluster';
+
+function ClusterConfigCard({ cluster }: { cluster: Cluster }) {
+  const { user } = useAuth();
+  const jar = cluster?.jars[cluster.jars.length - 1];
```

Comment:
cluster?.jars[cluster.jars.length - 1] might fail.

If cluster?.jars is undefined then [] will error out.

Reply:
Array.isArray(cluster?.jars) ? cluster.jars.at(-1) : null

might be better